CHANGES.rst

```diff
@@ -5,7 +5,7 @@
 4.5.1 (unreleased)
 ==================
 
-- Nothing changed yet.
+- Make the tests compatible with ``zope.i18n >= 4.5``.
```

Comment:
Does this restriction need to go in `setup.py`?

Reply:
I don't think so. In older versions the `translate` method is called with less parameters. As all parameters are optional this does not hurt.